viz/gov.noaa.nws.ncep.viz.overlays/plugin.xml

```diff
@@ -66,24 +66,6 @@
             </command>
             <menu
                    label="Plots">
```

Comment:
So, you should be able to remove this `<menu` section... and then just add a single menu item, like the pgen one right above, using a `<menuContribution` section.  And try the `locationURI="menu:tools?after=P"` and see if that puts it in the right place... if not, I can help you get it in the right place